### DISCUSSION TITLE
Changing the table alignment in the docx from left to center

### DISCUSSION THREAD

**Discussion by aburkov** [jgm/pandoc#9988] [Q&A]
Hi,

I'm converting from markdown to docx. I use a custom reference.docx. I have an issue with making the tables centered. I'm changing the style associated with the table in the reference.docx. The style I see is called Compact. I change it to center, but the rendered docx still has all tables aligned to the left. In the rendered docx the style of the table becomes Compact+Left for some reason
![Screenshot 2024-07-15 at 2 55 45 PM](https://github.com/user-attachments/assets/ab479c64-ae13-4439-9902-0e5d2e103c9a)
.

What else should I change to make it work?

I couldn't find the style with name Table and couldn't create one because Word says it's reserved for built-in styles.

Thanks!

**Comment by jgm** [comment]
That's exactly what you do -- modify the style named Table.  I don't know why you don't see it.  Are you using a recent version of pandoc?

**Comment by TomBener** [comment]
This [issue comment](https://github.com/jgm/pandoc/issues/3275#issuecomment-369198726) is probably helpful for you.

**Reply by aburkov** [reply]
> This [issue comment](https://github.com/jgm/pandoc/issues/3275#issuecomment-369198726) is probably helpful for you.

This worked! Thank you.
